src/components/atoms/SimpleInput/SimpleInput.js

```diff
@@ -1,15 +1,27 @@
-import React from 'react';
+import React, { useEffect, useRef } from 'react';
 import PropTypes from 'prop-types';
 import './SimpleInput.scss';
 
-const SimpleInput = ({ label, name, value, type, onChange }) => (
-  <div className="formField">
-    <div className="label-field" htmlFor={name}>
-      {label}*
+const SimpleInput = ({ label, name, value, type, onChange }) => {
+  const inputRef = useRef(null);
```

Comment:
It's great that we have another reusable component :) One more thing that we should do here is to make input focus conditional - we don't want to focus all inputs every time.

We can do it adding a non-required `focusOnRender` prop that will have `defaultProps` value `false` and focusing input conditionally:
```
 useEffect(() => {
   if (focusOnRender) {
     focusInput();
   }
  }, []);